std/builtin/base.neva

```diff
@@ -49,4 +49,11 @@ type EqElse<T any> struct {
 }
 
 #extern(int int_eq, float float_eq, string string_eq)
-pub component Eq<T int |float | string>(a T, b T) (then T, else EqElse<T>)
\ No newline at end of file
+pub component Eq<T int |float | string>(a T, b T) (then T, else EqElse<T>)
+
+// List
+#extern(length)
+pub component Length<T>(data T) (res int)
```

Comment:
Let's use overloading to support maps (you don't have to write runtime function, just the interface here)

```neva
#extern(list list_len, map map_len)
pub component Len<T list<any> | map<any>>(data T) (res int)
```